fieldsolver/ldz_magnetic_field.cpp

```diff
@@ -231,17 +244,48 @@ void propagateMagneticFieldSimple(
    timer=phiprof::initializeTimer("Compute cells");
    phiprof::start(timer);
    
-   #pragma omp parallel for collapse(3)
+   #pragma omp parallel for collapse(3) schedule(dynamic,1)
    for (int k=0; k<gridDims[2]; k++) {
       for (int j=0; j<gridDims[1]; j++) {
          for (int i=0; i<gridDims[0]; i++) {
 
             // Set the fsgrid rank in the technical grid
+            // FIXME Why is this done repeatedly here? Yann can't remember (20191119).
```

Comment:
Doesn't quite seem to make any sense. As far as I can grep, this is the only point where that value ever gets touched. Maybe we can just do away with it altogether?

Reply:
No we want to set it once somewhere for IO purposes. But maybe we can do that in datareducer.cpp if it's only used in IO?

Reply:
I moved that.